src/lib/__tests__/InstantSearch-test.js

```diff
@@ -101,6 +103,9 @@ See: https://www.algolia.com/doc/guides/building-search-ui/going-further/backend
   });
 
   it('throws if insightsClient is not a function', () => {
+    const warn = jest.spyOn(global.console, 'warn');
```

Comment:
Why do we need these changes here?

Reply:
a continuation of #4746, the warnings now take up a lot of place, so I removed them from this file.